frontend/apps/marketing/src/components/contentful/carousels/imageCarousel/ImageCarousel.tsx

```diff
@@ -42,7 +42,13 @@ const ImageCarousel: React.FC<ImageCarouselProps> = ({
 
           return {
             id: title,
-            slide: <Image src={file?.url} altText={description} />,
+            slide: (
+              <Image
+                src={file?.url}
```

Comment:
Non-blocking, but if there's an undefined image should the slide even exist? I know this is existing so OK to do as a followup (if needed).

Reply:
I can take a look at this during the Carousel refactor.